### ISSUE TITLE
The prepublish script is not cross-platform compatible

### ISSUE THREAD

**Issue by Pomax** [Pomax/react-onclickoutside#312]
The `prepublish`  script is currently of a form that cannot ever work on a windows machine: `test $(npm -v | tr . '\\n' | head -n 1) -ge '4' || exit 1` uses shell-specific syntax. If there is no cross-platform way to run this as a set of shell instructions, then there might be node projects to help turn this into something cross-platform compatible (e.g. `shx` and the like), or barring that, be turned into a `prepublish.js` that can be invoked using `"prepublish": "node test/prepublish.js"`.

/cc @Andarist

**Comment by Andarist** [comment]
Yeah, this would have to be rewritten to plain JS to achieve cross-platform compatibility in the easiest way.
